src/expipe_plugin_cinpla/scripts/process.py

```diff
@@ -244,14 +244,18 @@ def process_ecephys(
         if "group" not in sorting.get_property_keys():
             compute_and_set_unit_groups(sorting, recording_cmr)
 
+        sparsity_temp_folder = si_folder / "tmp"
+        if sparsity_temp_folder.is_dir():
+            shutil.rmtree(sparsity_temp_folder)
+
         we = si.extract_waveforms(
             recording_cmr,
             sorting,
             folder=output_base_folder / "waveforms",
             overwrite=True,
             ms_before=ms_before,
             ms_after=ms_after,
-            sparsity_temp_folder=si_folder / "tmp",
+            sparsity_temp_folder=sparsity_temp_folder,
```

Comment:
Currently, the `spikeinterface` version has been pinned to `>=0.100.0,<0.101.0`. For future reference, the new `0.101.1` version will trigger an assertion if `sparsity_temp_folder` is not `None`: https://github.com/SpikeInterface/spikeinterface/blob/main/src/spikeinterface/core/waveforms_extractor_backwards_compatibility.py#L81